opencga-core/src/main/java/org/opencb/opencga/core/models/job/JobRetryParams.java

```diff
@@ -6,14 +6,16 @@ public class JobRetryParams {
 
     private String job;
     private boolean force;
+    private String scheduledStartTime;
     private ObjectMap params;
 
     public JobRetryParams() {
     }
 
-    public JobRetryParams(String job, boolean force, ObjectMap params) {
+    public JobRetryParams(String job, boolean force, String scheduledStartTime, ObjectMap params) {
         this.job = job;
         this.force = force;
+        this.scheduledStartTime = scheduledStartTime;
```

Comment:
This should be a QueryParam instead of a BodyParam. Otherwise, we should move here the "description", "tags", "dependencies" and "priority"

Reply:
I don't agree here. This WS doesn't follow any of our API rules. For API consistency, the job id that needs to be retried should be passed as a path param (not as a body param), force should be a query param, and the rest of the parameters that will apply to the new generated job should be passed in the body (new id, new description, tags, scheduledStartTime...). In any case, I will move scheduledStartTime where you say so it is consistent with the rest of the parameters, but that's definitely not the place

Reply:
I understand what you're saying. My argument here was solely on the consistency with the rest of the method.